lib/galaxy/schema/schema.py

```diff
@@ -1261,18 +1254,18 @@ class HistorySummary(HistoryBase, WithModelClass):
 class HistoryActiveContentCounts(Model):
     """Contains the number of active, deleted or hidden items in a History."""
 
-    active: int = Field(
-        ...,
+    active: Optional[int] = Field(
```

Comment:
Hmm, is the `Optional` part required here ? Do we pass `None` explicitly here, and can we change the serialization to instead pass 0 ? That would seem like a `friendlier` type

Reply:
Very likely, this model was written 3 years ago and not used until now, so it probably needs some love. I think I tried to set the default value to 0 at some point but some tests or UI were failing IIRC. I'll recheck and see what I can do :+1:

Reply:
I modified the serializer to ensure it returns 0 instead of None and made those fields required again.